anta/models.py

```diff
@@ -448,15 +448,16 @@ def __init__(
         self.device: AntaDevice = device
         self.inputs: AntaTest.Input
         self.instance_commands: list[AntaCommand] = []
-        self.result: TestResult = TestResult(
-            name=device.name,
-            test=self.name,
-            categories=self.categories,
-            description=self.description,
-        )
+        self.result: TestResult = TestResult(name=device.name, test=self.name, categories=self.categories, description=self.description)
         self._init_inputs(inputs)
-        if self.result.result == AntaTestStatus.UNSET:
+        if hasattr(self, "inputs"):
             self._init_commands(eos_data)
+            if res_ow := self.inputs.result_overwrite:
+                if res_ow.categories:
+                    self.result.categories = res_ow.categories
+                if res_ow.description:
+                    self.result.description = res_ow.description
+                self.result.custom_field = res_ow.custom_field
```

Comment:
why is this moving? this should be part of the inputs still no?

Reply:
It is equivalent.
It is overriding the TestResult fields with some inputs.
Having move outside of "init_inputs" is more readable since it is not initiating any inputs, but using initiated inputs to modify the result.